plugins/modules/fabric_transits_workflow_manager.py

```diff
@@ -1262,6 +1300,15 @@ def main():
     # Create an AnsibleModule object with argument specifications
     module = AnsibleModule(argument_spec=element_spec, supports_check_mode=False)
     ccc_sda_transit = FabricTransit(module)
+    if ccc_sda_transit.compare_dnac_versions(ccc_sda_transit.get_ccc_version(), "2.3.7.6"):
```

Comment:
if ccc_sda_transit.compare_dnac_versions(ccc_sda_transit.get_ccc_version(), "2.3.7.6") >= 0:

We start to support form 2.3.7.6 ?

Reply:
You need to compare with >= 0 rite?